configs/exokit.json

```diff
@@ -2,7 +2,7 @@
   "index_name": "exokit",
   "start_urls": [
     "https://docs.webmr.io/docs/",
```

Comment:
![capture 12](https://user-images.githubusercontent.com/6926057/43419353-aae3ea68-940e-11e8-99b6-8326ad99eacf.PNG)

Reply:
fixed in https://github.com/algolia/docsearch-configs/pull/512/commits/e7d2901071ffe5248d1b2536c2856bae82f32916